client/rpc/block.go

```diff
@@ -2,12 +2,11 @@ package rpc
 
 import (
 	"fmt"
-	"net/http"
-	"strconv"
-
 	"github.com/gorilla/mux"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"net/http"
```

Comment:
Please move this back.

Reply:
What do you mean by that?

Reply:
Actually you just need to run goimport to reformat this.

Reply:
i.e run `make format`

Reply:
Sorry, was using gofmt instead of goimports. Fixed it now